config/settings/test.yml

```diff
@@ -445,7 +446,3 @@ va_notify:
 
 accredited_representative_portal:
   pilot_user_email_poa_codes:
-
-vaos:
```

Comment:
Duplicate on [line 263](https://github.com/department-of-veterans-affairs/vets-api/blob/b9a3e15058d4ca718b9242c769061588afa74a89/config/settings/test.yml#L263)